include-build/roborazzi-gradle-plugin/build.gradle

```diff
@@ -18,6 +18,7 @@ dependencies {
     // We don't use junit for plugin
     exclude group: 'junit', module: 'junit'
   }
+  implementation 'com.google.guava:guava:31.0.1-jre'
```

Comment:
Not sure if using guava is the best choice for generating file hashes. I roughly copied how to generate them from the gradle-checksum plugin: https://github.com/gradle/gradle-checksum/tree/master.